frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -348,6 +351,42 @@ export const getTopologyGroupItems = (dc: K8sResourceKind, groups: Group[]): Gro
   return groups;
 };
 
+export const getFilteredTrafficWorkload = (nodes: KialiNode[]): KialiNode[] =>
+  nodes.filter(({ data }) => data.nodeType === 'workload');
```

Comment:
I believe there is a type for this (I didn't know this when previously reviewing)...

```suggestion
  nodes.filter(({ data }) => data.nodeType === TYPE_WORKLOAD);
```

From `frontend/packages/dev-console/src/components/topology/const.ts`